tests/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/terraform-ibm-modules/terraform-ibm-module-template
+module github.com/terraform-ibm-modules/terraform-ibm-app-configuration
 
 go 1.20
```

Comment:
can you update this to go 1.21 and re-run `make go-mod-tidy` locally? (Ensure your locally on 1.21 before doing this)

Reply:
Also update the test wrapper version to the current latest as it has some important fixes